meta-facebook/gt-cc/prj.conf

```diff
@@ -0,0 +1,69 @@
+
+# cmsis-os2
+CONFIG_CMSIS_RTOS_V2=y
+CONFIG_NUM_PREEMPT_PRIORITIES=56
+CONFIG_HEAP_MEM_POOL_SIZE=32768
+CONFIG_CMSIS_V2_MSGQ_MAX_DYNAMIC_SIZE=8192
+CONFIG_POLL=y
+CONFIG_THREAD_NAME=y
+CONFIG_THREAD_STACK_INFO=y
+CONFIG_THREAD_MONITOR=y
+CONFIG_IRQ_OFFLOAD=y
+CONFIG_INIT_STACKS=y
+CONFIG_MAX_THREAD_BYTES=4
+CONFIG_SCHED_SCALABLE=y
+CONFIG_CMSIS_V2_MEM_SLAB_MAX_DYNAMIC_SIZE=128
+CONFIG_CMSIS_V2_THREAD_MAX_COUNT=23
+CONFIG_CMSIS_V2_THREAD_DYNAMIC_MAX_COUNT=10
+CONFIG_TIMEOUT_64BIT=n
+CONFIG_MP_NUM_CPUS=1
+
+# disable for qemu delay too short issue
+CONFIG_QEMU_ICOUNT=n
+
+# define aspeed driver
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_SPI_NOR_MULTI_DEV=y
+CONFIG_I2C=y
+CONFIG_THREAD_RUNTIME_STATS=y
+CONFIG_GPIO_SHELL=y
+CONFIG_I2C_SLAVE=y
+CONFIG_I2C_EEPROM_SLAVE=y
+CONFIG_I2C_IPMB_SLAVE=y
+CONFIG_PWM=n
+CONFIG_ADC=y
+CONFIG_JTAG=y
+CONFIG_FLASH=y
+CONFIG_FLASH_SHELL=y
+CONFIG_SENSOR=y
+CONFIG_HWINFO=y
+CONFIG_ESPI=n
+CONFIG_PECI=n
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=102400
+CONFIG_REBOOT=y
+CONFIG_POSIX_CLOCK=y
+CONFIG_STACK_SENTINEL=y
+CONFIG_LOG=y
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_SHELL=y
+CONFIG_SHELL_BACKEND_SERIAL_INIT_PRIORITY=99
+CONFIG_IPMI=y
+CONFIG_IPMI_KCS_ASPEED=n
+CONFIG_I3C=n
+CONFIG_I3C_SLAVE=n
+CONFIG_I3C_SLAVE_MQUEUE=n
+CONFIG_WATCHDOG=y
+#USB
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM"
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_CDC_ACM_RINGBUF_SIZE=576
+CONFIG_USB_DEVICE_VID=0x1d6b
+CONFIG_USB_DEVICE_PID=0x0104
+CONFIG_LOG_BUFFER_SIZE=5120
+#USB
+
+CONFIG_USERSPACE=y
```

Comment:
This config is to follow the settings that are enabled on other platforms (yv35-cl, yv35-bb).
The necessity of this setting is currently being discussed with Wiwynn and Aspeed, and it will be turned off if it is found not to be used in the future.